en_US/data-integration/snowflake.md

```diff
@@ -425,7 +425,6 @@ Before configuring the Snowflake Sink, you must create a connector in EMQX to es
 
    - **Account**: Enter your Snowflake Organization ID and Snowflake account name separated by a dash (`-`), which is part of the URL you use to access the Snowflake platform and can be found in your Snowflake console.
    - **Server Host**: The server host is the Snowflake endpoint URL, typically in the format `<Your Snowflake Organization ID>-<Your Snowflake Account Name>.snowflakecomputing.com`. You need to replace `<Your Snowflake Organization ID>-<Your Snowflake Account Name>` with the subdomain specific to your Snowflake instance.
-   - **Username**: Snowflake user with a registered RSA public key (e.g. `snowpipeuser`, as defined during the previous setup process).
```

Comment:
This is still a connector field.  It's just optional in case the user configures it in `odbc.ini`.